UpSkill_App/UpSkill.Infrastructure/Data/Models/LiveSession.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace UpSkill.Infrastructure.Data.Models
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+
+    public class LiveSession
+    {
+        [Key]
+        [Required]
+        public string Id { get; init; } = Guid.NewGuid().ToString();
+
+        [Required]
+        public string Topic { get; set; }
+
+        public decimal Price { get; set; }
+
+        public DateTime StartOfSession { get; set; }
+
+        public DateTime EndOfSession { get; set; }
+
+        public TimeSpan Duration { get; set; }
+
```

Comment:
If we have start and end time, the duration might not be necessary?

Reply:
I was just following the Wiki file, the contents of which we discussed on Saturday.